### PR TITLE
Add Twilio

### PR DESCRIPTION
**Pull request by setu4993**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by setu4993** [comment]
@conda-forge/staged-recipes, @conda-forge/help-python : This PR is ready for review.

File: recipes/twilio/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+    - six
+
+test:
+  imports:
+    - twilio
+    - twilio.base
+    - twilio.http
+    - twilio.jwt
+    - twilio.jwt.access_token
+    - twilio.jwt.client
```
Comment (marcelotrevisani) [inline recipes/twilio/meta.yaml]: It is not necessary to import all the modules and submodules. Testing just the basic modules should be fine. But if you want to you can let them here, you will just have more work when the modules e submodules changed from version to version

File: recipes/twilio/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+    - twilio.rest.wireless
+    - twilio.rest.wireless.v1
+    - twilio.rest.wireless.v1.sim
+    - twilio.twiml
+
+about:
+  home: "https://github.com/twilio/twilio-python/"
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.md
```
Comment (marcelotrevisani) [inline recipes/twilio/meta.yaml]: I opened a PR to this project to also package the license 
https://github.com/twilio/twilio-python/pull/489

File: recipes/twilio/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+  sha256: e78a2006b9449fb9fad5050537e0998c181c7d3a62eaa9eed434e59dbaf58324
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
```
Comment (marcelotrevisani) [inline recipes/twilio/meta.yaml]: in the ``host`` section I think you just need ``pip``, ``python`` and ``setuptools``

Comment (setu4993) [reply recipes/twilio/meta.yaml]: Thanks, only `pip` and `python` seem enough.

File: recipes/twilio/meta.yaml (hunk 2 above)
Comment (setu4993) [reply recipes/twilio/meta.yaml]: This is great! Once that is merged, can refer to the license file in the original package.

File: recipes/twilio/meta.yaml (hunk 1 above)
Comment (setu4993) [reply recipes/twilio/meta.yaml]: Good point, hadn't thought of that. I used what the `skeleton` generated as is, which came from the `twilio` tests module, I think. Removed a lot of them in the latest commit to leave out primary submodules.

**Pull request merged**